sylvain-pierrot/seasonal-workers
# feat: migrate http service to nats service

## What does this PR do ?
This PR aims to migrate the old Spring Boot HTTP microservice into a new NATS client microservice.

### How should this be manually tested ?

First, make sure you have the API gateway running (npm run start:dev). You can then test the service by executing the command `gradlew bootrun` from the profile directory and test the endpoints using [Postman](https://www.postman.com/winter-shadow-181910/workspace/seasonal/collection/524490-c849f4b4-e193-4477-bb6f-2b06c8102deb?action=share&creator=28680159).

### Other changes
